example-get-started-experiments/code/src/train.py

```diff
@@ -55,17 +56,25 @@ def train():
     if params.train.arch not in model_names:
         raise ValueError(f"Unsupported model, must be one of:\n{model_names}")
 
-    learn = unet_learner(
-        data_loader, arch=getattr(models, params.train.arch), metrics=DiceMulti
-    )
+    with Live("results/train") as live:
+        learn = unet_learner(
+            data_loader, arch=getattr(models, params.train.arch), metrics=DiceMulti
+        )
 
-    learn.fine_tune(
-        **params.train.fine_tune_args,
-        cbs=[DVCLiveCallback(dir="results/train", report="md")],
-    )
-    models_dir = Path("models")
-    models_dir.mkdir(exist_ok=True)
-    learn.export(fname=(models_dir / "model.pkl").absolute())
+        learn.fine_tune(
+            **params.train.fine_tune_args,
+            cbs=[DVCLiveCallback(dir="results/train", report="md")],
```

Comment:
```suggestion
            cbs=[DVCLiveCallback(live=live)],
```
https://dvc.org/doc/dvclive/ml-frameworks/fastai#examples